core/ui/UIRichText.h

```diff
@@ -254,6 +264,7 @@ class AX_GUI_DLL RichElementImage : public RichElement
     float _scaleX;
     float _scaleY;
     std::string _url; /*!< attributes of anchor tag */
+    std::string_view _id;
```

Comment:
Use `std::string` for storage?

Reply:
Sorry, I completely missed that. Changing it now.